### DISCUSSION TITLE
Kerberos on Windows

### DISCUSSION THREAD

**Discussion by skbrtzyb** [FreeRDP/FreeRDP#9939] [Q&A] [answered]
I am trying to build FreeRDP with Kerberos support. This is no problem on Linux, but I'm stuck with the Windows build.

I understand that FreeRDP does not use the Windows native Kerberos client but needs either a Heimdal or MIT library. Those that I found until now do not seem to be suitable to add to the build; they are e.g. missing a krb5_config file, required by the build script. I tried to hack my way around that with limited success. It ultimately failed on unresolved externals, indicating that I had the wrong version of the library.

So, I need some pointers to get started. Those of you who have built FreeRDP with Kerberos support successfully in Windows, what libraries did you use? Which versions? Where to find them? Are there any steps that I should take that I may have missed?

TIA,
Jeroen

**Comment by akallabeth** [comment, marked as answer]
no.
we use `WinPR` to emulate the windows `SSPI API` on linux (which uses `MIT` or `Heimdal`)
on windows this wrapper is not required and should be handled without anything else. (your mileage may vary, not very often tested, but feedback very welcome)

**Reply by skbrtzyb** [reply]
That's great. So I assume I also don't have to pass the WITH_KRB5 build flag, as that one causes the build scripts to complain about missing libraries?

**Comment by hardening** [comment]
IIRC with default options under windows you'll use the native SSPI of windows so no need for krb5 libs.

**Reply by skbrtzyb** [reply]
Thanks
